### DISCUSSION TITLE
138

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#814] [Comments]
http://jojoldu.tistory.com/138

**Comment by jojoldu** [comment]
작성자: 지나가는애

적녹색약이시네요
잘보고갑니다

**Comment by deepweller** [comment]
잘봤습니다.

**Comment by dncgr8** [comment]
안녕하세요 지금 스프링 부트 공부를 막 시작했는데 인텔리 제이에서 command +N 으로 새로운 파일 추가할때
어떤 경우는 후보군에 directory가 나오고 어떤때는 package가 나오는데 이유를 여쭤봐도 될까요_?
인텔리제이강의에서 생성가능한 후보들만 표시된다고 하신거 같은데...둘 차이를 몰라서 이렇게 헷갈리는건지 ..
알려주심 큰 도움이 될것 같습니다 !!!
